.gitignore

```diff
@@ -28,6 +28,17 @@
 /ui/src/generated/*
 /ui/src/notebooks-generated/*
 /ui/src/publicGenerated/*
+/public-ui/package-lock.json
+/public-ui/coverage
+/public-ui/ui.iml
+/public-ui/.idea/*
+/public-ui/**/*.js
+/public-ui/**/*.map
+/public-ui/dist/**
+/public-ui/yarn-error.log
+/public-ui/src/generated/*
```

Comment:
There's a bunch of references to this "generated" directory. AFAICT you don't populate it in your codegen scripts (just publicGenerated).

Reply:
Thanks. Good catch. Removed.